shared/node.go

```diff
@@ -126,6 +126,7 @@ func (m *Node) GetBus() modules.Bus {
 	return m.bus
 }
 
+// TECHDEBT: The `shared` package has dependencies on types in the individual modules.
```

Comment:
N.A.: Right, I guess that's why I added messages in `messaging` instead of the various packages.